.github/workflows/deploy-action.yaml

```
@@ -23,6 +23,11 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: npm i
     - run: npm run build
+    - name: Load Config JS
+      env:
+        FIREBASE_CONFIG_BASE64: ${{ secrets.TRENDLY_FIREBASE_CONFIG }}
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting

(shellcheck)